src/components/ProtectedRoute.tsx

```
@@ -1,26 +1,67 @@
-import { ReactNode } from 'react'
+import React, { ReactNode } from 'react'
 import { Navigate } from 'react-router-dom'
-import { useAuthContext } from '@/hooks/useAuthContext'
-import { Spinner, Center } from '@chakra-ui/react'
+import { useAuth } from '@/hooks/useAuth'
+import { usePermissions } from '@/hooks/usePermissions'
```

ESLint

[error] 'usePermissions' is defined but never used.

(@typescript-eslint/no-unused-vars)